registry/polkadot.json

```diff
@@ -334,4 +334,11 @@
         "longDescription": "**metaspan** is a trusted network infrastructure provider and validator on Polkadot and Kusama.\n\nWe are a team of experienced engineers and developers with a passion for blockchain technology and the Polkadot ecosystem.\n\nWe are actively involved in the community and contribute to the ecosystem via development, utilities, education, and advocacy.\n\nWe are committed to the long-term success of Polkadot and Kusama and are excited to be a part of the journey.",
         "isOrganization": true
     }
+    {
+        "address": "12eMZTAnXEsyedXmsB6jDVRnF9Mq8ZrhLefRGhxPE4JwrPAS",
+        "name": "MAD CRIPTO",
+        "image": "https://raw.githubusercontent.com/MADCRIPTO/opengov-delegate-registry/master/images/madcripto.png",
+        "shortDescription": "Activo en la governanza por el amor a Polkadot",
+        "longDescription": "Creo contenido en español y represento a la comunidad hablahispana en la governanza.",
+        "isOrganization": false
```

Comment:
Please, check the closures. Last one seems missing.

Reply:
Thanks, added closure.